pkgs/development/libraries/libspatialaudio/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, stdenv, fetchFromGitHub, cmake, libmysofa, zlib }:
+stdenv.mkDerivation {
+  name = "libspatialaudio";
+  src = fetchFromGitHub {
+    owner = "videolabs";
+    repo = "libspatialaudio";
+    rev = "0.3.0";
+    hash = "sha256-sPnQPD41AceXM4uGqWXMYhuQv0TUkA6TZP8ChxUFIoI=";
+  };
+  buildInputs = [ cmake libmysofa zlib ];
+  configurePhase = "cmake -DCMAKE_INSTALL_PREFIX=$out .";
+
+  meta = {
+    description =
+      "Ambisonic encoding / decoding and binauralization library in C++";
+    homepage = "https://github.com/videolabs/libspatialaudio";
+    license = lib.licenses.lgpl21Plus;
```

Comment:
Can you add yourself as a maintainer :thinking: ?

Reply:
Ok :) I've subscribed to updates to their repo.